### DISCUSSION TITLE
https://education.github.com/discount_requests is redirected.

### DISCUSSION THREAD

**Discussion by Kobe972** [community/community#131668] [GitHub Education]
### Hi everyone,

In https://education.github.com/pack/offers, after clicking on "I am a student", the page is redirected to https://github.com/pricing instead of https://education.github.com/discount_requests.

I opened a ticket and it is totally answered by robots, and then closed. I cannot reopen it or submit a new ticket (it says submitted successfully but I can't find the submitted ticket). I wish a HUMAN to answer the problem.

**Comment by TheGiraffe3** [comment]
Try contacting support@github.com or looking at support.github.com, though I'm not sure what the latter one does.

**Comment by davevad93** [comment]
Hi @Kobe972 ,

You should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue. Someone from GitHub Staff will answer you there, although it may take a while, so be patient.

**Reply by Kobe972** [reply]
After that ticket was closed, I can't open a new ticket. I submitted the ticket without any error but I can't see the new ticket in the ticket page.
